### DISCUSSION TITLE
Consolidating multiple hotspots?

### DISCUSSION THREAD

**Discussion by jakiejj** [Ysurac/openmptcprouter#3107] [Q&A]
I understand that openmptcprouter works with multiple WANS through ethernet ports. What if the WANs are all hotspots without rj45 ports, can openmptcprouter still work?

**Comment by iambenmitchell** [comment]
> I understand that openmptcprouter works with multiple WANS through ethernet ports. What if the WANs are all hotspots without rj45 ports, can openmptcprouter still work?

Yes. But you'd need to have a WiFi card for each of the hotspots to add them. 

Your experience might be bad though due to the extra latency WiFi introduces.

**Comment by jakiejj** [comment]
really need one wifi card per hotspot? I am no expert, but I think openwrt routers can act as a client-repeater. Can openmptcprouter be configured to do so similarly, like creating virtual interfaces acting as clients for the hotspots?

**Reply by iambenmitchell** [reply]
I'm not aware of any WiFi chipset that can do what is essentially virtualisation. Pretty sure the WiFi spec doesn't have anything like that. 

If you're bonding 2 connections you'd need 2 WiFi cards, 3 connections need 3 etc. 

With all the latency and unpredictability of WiFi.. I'd never bother doing that. 

Why do you need to use WiFi? If you device only has one Ethernet port, I'd recommend using a smart switch (vlan switch) and using vlans. 

If you must use WiFi, then perhaps USB to WiFi adapters would work for you? You could also do something even stupider like buying 3 or 4 raspberry pis and bridging the WiFi to their ethernets and then using that instead of WiFi cards.. but this is expensive and a terrible idea in my opinion. 

If you want to share why you need to be using WiFi, maybe I can give you a battery solution

**Reply by iambenmitchell** [reply]
If the reason you need to use WiFi is because you're tethering multiple phones, OMR supports iPhone tethering over USB natively. I imagine it also works the same with android too.

**Reply by jakiejj** [reply]
I am looking at some cheat hotspots like this: https://www.amazon.com/T-Mobile-Portable-Hotspot-Connect-Devices/dp/B0CD9PMGQQ

it does look like having a usb port, not sure if usb tethering works on those. hmm interesting, thanks!

**Reply by iambenmitchell** [reply]
You used to be able to buy USB SIM card dongles. Essentially what you've sent but with a much smaller package without WiFi. 

Not sure if TMobile sell them but you can get them on Amazon I'm sure and then just put a sim in it. Not sure what the speed of them are these days. https://www.t-mobile.com/support/devices/mobile-internet/usb-laptop-sticks/device-hardware-t-mobile-rocket-4g-usb-laptop-stick

**Reply by jakiejj** [reply]
As for the router acting as a wifi-client and a wifi-server, I was referring to this: https://www.gargoyle-router.com/wiki/doku.php?id=bridge_repeater
